platform/darwin/src/NSData+MGLAdditions.h

```diff
@@ -0,0 +1,13 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Headers that use `NS_ASSUME_NONNULL_BEGIN` should also import MGLTypes.h, or Xcode 6 will fail to compile the code. We’re considering phasing out this requirement, however: #5472.